lib/ecto/migration/runner.ex

```diff
@@ -16,7 +17,12 @@ defmodule Ecto.Migration.Runner do
   """
   def run(repo, module, direction, operation, migrator_direction, opts) do
     level = Keyword.get(opts, :log, :info)
-    start_link(repo, direction, migrator_direction, level)
+    ensure_manager_started()
```

Comment:
We should always start the manager when Ecto starts! This can be done in two steps:

1. Adding `mod: {Ecto, []}` to `def application` in `mix.exs` so it invokes the callback in the Ecto module

2. By adding `use Application` in Ecto and returning a supervision tree where the manager is the only child

3. From there on, we should just assume the manager is available and running :)